README.md

```diff
@@ -1,15 +1,22 @@
 # ISLE Fedora
 
+## Part of the ISLE Islandora 7.x Docker Images
+Designed as the Fedora Repository for ISLE.
+
 Based on:  
  - [ISLE-Tomcat](https://github.com/Islandora-Collaboration-Group/isle-tomcat)
- - Fedora Repository 3.8.1
- - @todo: complete me
 
-# Generic Usage
+Contains and Includes:
+ - [Fedora Repository 3.8.1](https://duraspace.org/fedora/)
+ - [Gsearch 2.9](https://github.com/discoverygarden/gsearch.git) (_DGI fork_)
+ - [Maven 3.6.0](https://maven.apache.org/)
```

Comment:
This needs to be changed to 3.6.1
Ant needs to be changed to 1.10.6
I would also be fine with leaving maven and ant out of this list.

Reply:
I think we should leave on the list as a pre-caution given their importance in not only building the Dockerfile but they can be used by ISLE users for building other configurations.